content/en/platform/corda/5.0-dev-preview-2/developing/api/api-persistence.md

```diff
@@ -8,3 +8,173 @@ menu:
     weight: 7000
 section_menu: corda-5-dev-preview
 ---
+Corda 5 supports CRUD (Create, Read, Update, Delete) operations for user-defined types. This is achieved using JPA annotated entities and, to manage DB migrations, Liquibase.
+
+## Defining custom tables using Liquibase migrations
```

Comment:
There no number steps in this task so sometimes I'm unsure who's doing what, for example, "associated tables, indexes, foreign-keys, etc. must be created." - created by the system or by you, the user? "This is done by embedding Liquibase files into the CorDapp [CPK]" - Am I doing it now as a user?
Maybe it's worth considering adding some steps or active voice or be a bit more specific in some places. Or maybe it's just obious for a dev reading it and I'm not understanding anything ;-)